sideboard/tests/test_sa.py

```diff
@@ -9,31 +9,14 @@
 from sqlalchemy.ext.hybrid import hybrid_property
 from sqlalchemy.orm import relationship
 from sqlalchemy.types import Boolean, Integer, UnicodeText
-from sqlalchemy.schema import Column, ForeignKey, UniqueConstraint
+from sqlalchemy.schema import Column, CheckConstraint, ForeignKey, MetaData, Table, UniqueConstraint
 from sqlalchemy.sql import case
 
 from sideboard.lib import log, listify
 from sideboard.tests import patch_session
 from sideboard.lib.sa._crud import normalize_query, collect_ancestor_classes
-from sideboard.lib.sa import SessionManager, UUID, JSON, declarative_base, CrudException, crudable, text_length_validation, regex_validation
-
-
-def test_declarative_base_without_parameters():
-
-    @declarative_base
-    class BaseTest:
-        pass
-
-    assert BaseTest.__tablename__ == 'base_test'
-
-
-def test_declarative_base_with_parameters():
```

Comment:
I just moved these into `TestDeclarativeBaseConstructor ` which is a more appropriate location.